pkgs/by-name/bp/bpfilter/package.nix

```diff
@@ -0,0 +1,77 @@
+{
+  bison,
+  cmake,
+  cmocka,
+  doxygen,
+  fetchFromGitHub,
+  flex,
+  lcov,
+  lib,
+  libbpf,
+  libnl,
+  nix-update-script,
+  pkg-config,
+  python3,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bpfilter";
+  version = "0.2.0";
```

Comment:
There’s already a 0.2.1 release. Any reason to stick with 0.2.0?

Reply:
nope, I just missed it! thanks :)